helpers/gather-os-metrics.js

```diff
@@ -0,0 +1,40 @@
+/* eslint no-console: "off" */
+
+const pidusage = require('pidusage');
+const os = require('os');
+const sendMetrics = require('./send-metrics');
+
+module.exports = (io, span) => {
+  const defaultResponse = {
+    2: 0,
+    3: 0,
+    4: 0,
+    5: 0,
+    count: 0,
+    mean: 0,
+    timestamp: Date.now(),
+  };
+
+  pidusage.stat(process.pid, (err, stat) => {
+    if (err) {
+      console.error(err);
+      return;
```

Comment:
That's a very good idea @soyuka at the moment I just moved the method inside another file and haven't touched it yet; will keep you in the loop when I will add tests that will cover this scenario